src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -288,6 +288,13 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('role')->defaultValue('ROLE_ALLOWED_TO_SWITCH')->end()
                 ->end()
             ->end()
+            ->arrayNode('session_expiration')
+                ->canBeUnset()
+                ->children()
+                    ->integerNode('max_idle_time')->defaultValue(ini_get('session.gc_maxlifetime'))->min(1)->end()
```

Comment:
I guess using `null` here and determine the ini setting value at runtime then would be safer.